lib/LANraragi/Plugin/Metadata/Eze.pm

```diff
@@ -79,8 +85,43 @@ sub get_tags {
             return ( tags => $tags );
         }
 
+    } elsif (-e $path_nearby_json){
```

Comment:
Don't duplicate the entire codeblock here, just create a new sub and call it both here and in the `path_in_archive` case.  
```
elsif (-e $path_nearby_json){
 
$logger->debug("Found file $path_nearby_json");
my $hashjson = consume_json($path_nearby_json);

#Parse it
my ( $tags, $title ) = tags_from_eze_json($hashjson);
[...]
}

sub consume_json() {
     my $filepath = $_[0];

     #Open it
     my $stringjson = "";
     [...]
     $logger->debug("Found nearby and loaded the following JSON: $stringjson");
     return $hashjson;
}
```